addons/binding/org.openhab.binding.milight/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,118 @@
+        <description>This bridge represents a Milight device.</description>
+        
+        <config-description>
+            <parameter name="ADDR" type="text" required="true">
+                <label>IP or Host Name</label>
+                <description>Will be resolved by discovery if auto configured
+                </description>
+            </parameter>
+            <parameter name="PORT" type="integer" required="true">
+                <label>Bridge port</label>
+                <description>This differs for different bridge devices. The newest ones use 8899. Older ones use 50000.
+                </description>
+                <default>8899</default>
+            </parameter>
+            <parameter name="ID" type="text" required="false">
+                <label>Bridge ID</label>
+                <description>The mac address in upper case letters without delimiter for Version 3+ bridges. Leave empty for Version 1/2 devices.
+                This is used to check if the given IP corresponds to the right device. The bridge is set offline if the device
+                does not respond with the correct Bride ID and a re-detection is started. Useful for DHCP environments where
+                IPs may change over time, after power outage etc.
+                </description>
+                <advanced>true</advanced>
+            </parameter>
+            <parameter name="REFRESH" type="integer" min="1" max="60">
+                <label>Refresh interval</label>
+                <description>Interval in minutes to check for device presence. The Bridge ID is used to check if the IP is still the right one.
+                </description>
+                <default>2</default>
+                <advanced>true</advanced>
+            </parameter>
+        </config-description>
+	</bridge-type>
+	
+    <thing-type id="rgbLed">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="bridge" />
+        </supported-bridge-type-refs>
+        <label>RGB Led</label>
+        <description>Milight RGB Led</description>
+
+        <channels>
+            <channel id="ledcolor" typeId="ledcolor"/>
+            <channel id="discospeed" typeId="discospeed"/>
+            <channel id="discomode" typeId="discomode"/>
+        </channels>
+    </thing-type>
+    
+    <thing-type id="whiteLed">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="bridge" />
+        </supported-bridge-type-refs>
+        <label>White Led</label>
+        <description>Milight White Led</description>
+
+        <channels>
+            <channel id="ledbrightness" typeId="ledbrightness"/>
+            <channel id="ledtemperature" typeId="ledtemperature"/>
+            <channel id="discospeed" typeId="discospeed"/>
+            <channel id="discomode" typeId="discomode"/>
+        </channels>
+    </thing-type>
+
+
+    <channel-type id="ledcolor">
+        <item-type>Color</item-type>
+        <label>Farbe</label>
```

Comment:
why do you suddenly switch to German here? The default language should always be English, please!